docs/user_guide/perf_analyzer.md

```diff
@@ -196,6 +196,38 @@ components of the latency. Follow these steps:
 - Select "Upload" and upload the file
 - Select "Replace data at selected cell" and then select the "Import data" button
 
+### Server-side metrics
+
+Perf Analyzer can optionally collect and output server-side metrics, such as GPU
+utilization and GPU power usage. To enable this collection/output, use the
+`--verbose-csv` (with the `-f <filename>` option) and `--collect-metrics`
```

Comment:
I would split up talking about collection (--collect-metrics) vs output to csv since soon it will also output to stdout.  Also the csv stuff should go after talking about the metrics-url option

Reply:
Rearranged 👍